swift/ios-shoppe-demo/Extensions/EX+UIView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  EX+UIView.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 3/30/20.
+//  Copyright © 2020 Harold Davis Jr. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
+    func curve() {
```

Comment:
The `curve()` and `smallCurve()` are similar.  Would it be better to do something like `curve(cornerRadius, shadowRadius)` and then allow the caller to pass these in?  These curves are also adding shadows, so they do more than just curve.  If we changed this to only handle `cornerRadius` then it would handle all 3 cases.  I wonder if we might also be able to return `self` at the end of this and create a decorator pattern. Then you could so `view.curve(20).shadow()` assuming we create a `shadow()` extension.

Reply:
Is it ok I keep the shadow extension in mind for later? I did upgrade the parameters and the function name so its more flexible, I also got rid of the others to curveViewCornersWithShadow(cornerRadius: Float? = nil, shadowOffset: CGSize? = nil, shadowOpacity: Float? = nil, shadowRadius: Float? = nil)